### PR TITLE
UI is now more complete

### PR DESCRIPTION
**Pull request by kognate**
Create a new classifier based on user uploads

This creates a new classifier based on user uploads but does not support drag and drop.  The test data I picked at random from the other bundles.

### CODE REVIEW

**Comment by nfriedly** [comment]
In testing this locally, I uploaded some dog .zips and then tried the last image (the golden retriever, which should have been a hit), and got this:

```
TypeError: mapToCheck[kind] is undefined
lookupInMap() bundle.js:768
renderTable/classesModel</classes<() bundle.js:887
map() self-hosted:232
renderTable/classesModel<() bundle.js:885
renderTable() bundle.js:884
showResult() bundle.js:603
...
```

**Comment by kognate** [comment]
I'm looking at that.

File: public/js/use.js [hunk 1]
```diff
@@ -280,6 +290,84 @@ function setupUse(params) {
...
+    console.log(image.getBoundingClientRect(), ratio, image.getBoundingClientRect().width, faceLocation.width);
+    return newFaceLocation;
+  }
+
+  /*
+   * Solution found here:
+   * http://stackoverflow.com/questions/5598743/finding-elements-position-relative-to-the-document#answer-26230989
+   */
+  function getCoords(elem) { // crossbrowser version
+    var box = elem.getBoundingClientRect();
```
Comment (nfriedly) [inline public/js/use.js]: Is this any different from `jQuery(elm).offset();` ?  http://api.jquery.com/offset/

**Comment by nfriedly** [comment]
Confirmed that the first bug was fixed. Everything seems to be working now. Did you say there was one more checking coming in @kognate, or is this ready to merge in now?

**Comment by kognate** [comment]
There is one more thing.

File: views/includes/train.jade [hunk 2]
```diff
@@ -27,6 +27,51 @@
     ._training--example.clickable(data-kind='omniearth')
       h3 Satellite Imagery
       img(src='/images/bundles/omniearth/thumbnail.png')
+    ._training--example.clickable(data-kind='user')
```
Comment (nfriedly) [inline views/includes/train.jade]: Ok, I guess some of the templates are real jade

File: views/includes/underscore-templates.jade [hunk 3]
```diff
@@ -138,3 +138,7 @@ script(type='text/html', id='useResultsTableTemplate').
       </tbody>
     </table>
   </div>
+script(type='text/html', id='imageBoxesTemplate').
+  <% _.each(items, function(item, key, list) { %>
+    <div class="box-location" style="height: <%= item.height %>px; left: <%= item.left %>px; top: <%= item.top %>px; width: <%= item.width %>px;"></div>
+  <% }) %>
```
Comment (nfriedly) [inline views/includes/underscore-templates.jade]: but not this one ;)

**Pull request merged**